tests/BenchmarkDotNet.IntegrationTests/InProcessTest.cs

```diff
@@ -217,6 +217,32 @@ public void InProcessBenchmarkAllCasesSupported()
             }
         }
 
+
+#if NET8_0_OR_GREATER
+        [Fact]
+        public void ParamsSupportRequiredProperty()
+        {
+            var logger = new OutputLogger(Output);
+            var config = CreateInProcessConfig(logger);
+            var summary = CanExecute<ParamsTestRequiredProperty>(config);
+            var log = logger.GetLog();
+
+            foreach (var param in new[] { "a", "b" })
+            {
+                Assert.Contains($"| Benchmark | {param}             |", log);
+            }
+        }
+
+        public class ParamsTestRequiredProperty
+        {
+            [Params("a", "b")]
+            public required string ParamProperty { get; set; }
+
+            [Benchmark]
+            public void Benchmark() => Console.WriteLine($"// ### New Parameter {ParamProperty} ###");
```

Comment:
I think asserting in the benchmark method is cleaner, no need to parse the log. Something like this:

```suggestion
            const string expected = "a";
            
            [Params(expected)]
            public required string ParamProperty { get; set; }

            [Benchmark]
            public void Benchmark() => Assert.Equals(expected, ParamProperty);
```